sportngin/hubstats
# Upgrading Hubstats to Rails 4.2.5

Description and Impact
----------------------
Hopefully no change will be actually visible. However, this is completely upgrading the version of Rails, so....

Deploy Plan
-----------
Nothing unique for the "deploy plan". Make a new "major" tag release for this PR when it's merged into master.

Rollback Plan
-------------
* To roll back this change, revert the merge with: `git revert -m 1 MERGE_SHA` and perform another deploy.

QA Plan
-------
* Just look over all of the changes, make sure that nothing looks wonky, and then let's try to maybe actually use it on a staging environment.
